### DISCUSSION TITLE
V3 - versioning functionality

### DISCUSSION THREAD

**Discussion by user7024** [dotnet/docfx#8174] [Q&A]
Hi,

A similar question has already [been answered](https://github.com/dotnet/docfx/discussions/7976), however, it was half a year ago. I have tested v3 beta version of DocFX, and I see that the versioning functionality doesn't currently work as specified in design documentation and specs. Do I get it right that this functionality is yet to be implemented? Because I'm trying to figure out whether the functionality is not yet available or it is expected to work to some extent, and it's just me doing something wrong with the configuration.

Thank you.

**Comment by markjulmar** [comment]
Yes, you are correct -- V3 breaks a bunch of features available in V2. The project maintainers are looking at pulling some of the more useful features of V3 back into the main branch but we're not going forward with V3 so I don't think it will ever be implemented/fixed in that branch.
